diff --git a/src/main/java/vendored/com/jeff_media/customblockdata/CustomBlockData.java b/src/main/java/vendored/com/jeff_media/customblockdata/CustomBlockData.java
index 67abdb6..1d4e95d 100644
--- a/src/main/java/vendored/com/jeff_media/customblockdata/CustomBlockData.java
+++ b/src/main/java/vendored/com/jeff_media/customblockdata/CustomBlockData.java
@@ -85,7 +85,6 @@ public class CustomBlockData implements PersistentDataContainer {
         PersistentDataType.BYTE_ARRAY,
         PersistentDataType.INTEGER_ARRAY,
         PersistentDataType.LONG_ARRAY,
-        PersistentDataType.TAG_CONTAINER_ARRAY,
         PersistentDataType.TAG_CONTAINER
       };
 
@@ -206,7 +205,6 @@ public class CustomBlockData implements PersistentDataContainer {
             .getLogger()
             .warning(
                 "Nag author(s) "
-                    + String.join(", ", plugin.getDescription().getAuthors())
                     + " of plugin "
                     + plugin.getName()
                     + " for not relocating the CustomBlockData package.");
