diff --git a/src/main/java/vendored/com/technicjelle/BMUtils/Cheese.java b/src/main/java/vendored/com/technicjelle/BMUtils/Cheese.java
index 25ed4cc..58a7561 100644
--- a/src/main/java/vendored/com/technicjelle/BMUtils/Cheese.java
+++ b/src/main/java/vendored/com/technicjelle/BMUtils/Cheese.java
@@ -33,6 +33,7 @@ import java.util.TreeMap;
 /**
  * A {@link Shape} with possibly some holes.
  */
+@SuppressWarnings({"all"})
 public class Cheese {
   private static final Vector2d CHUNK_CELL_SIZE = Vector2d.from(16, 16);
 
